Makefile

```diff
@@ -65,23 +65,11 @@ deps: $(BUILD_DEPS)
 .PHONY: deps
 
 build-devnets: build lotus-seed lotus-shed lotus-wallet lotus-gateway
-.PHONY: build-devnets
+.PHONY:build-devnets
 
-debug: GOFLAGS+=-tags=debug
+debug: GOFLAGS += -tags=debug
 debug: build-devnets
 
-2k: GOFLAGS+=-tags=2k
-2k: build-devnets
-
-calibnet: GOFLAGS+=-tags=calibnet
-calibnet: build-devnets
-
-butterflynet: GOFLAGS+=-tags=butterflynet
-butterflynet: build-devnets
-
-interopnet: GOFLAGS+=-tags=interopnet
-interopnet: build-devnets
```

Comment:
We may want to keep those for compat for now - just changing the default network in the binary.